src/angular-app/languageforge/lexicon/editor/field/dc-entry.component.html

```diff
@@ -13,7 +13,7 @@
             <div class="comment-bubble-entry">
                 <comment-bubble control="$ctrl.control" field="$ctrl.fieldName" model="$ctrl.model" parent-context-guid="$ctrl.contextGuid"></comment-bubble>
             </div>
-            &nbsp;Entry
+            &nbsp;Entry, Number: {{$ctrl.entryIndex}}
```

Comment:
Is this addition intentional?

Reply:
This was discussed in Slack, Alex liked this suggestion to add the entry number to the card.
https://sil-language-software.slack.com/archives/CGDKZ7NTV/p1633621109496300